### DISCUSSION TITLE
Using a channel on IGEL Community on Slack vs. GitHub Discussions

### DISCUSSION THREAD

**Discussion by rneher-igel** [falkheiland/PSIGEL#30] [General]
I'm thinking it might be better to have a channel on IGEL Community on Slack for PSIGEL (keep the traffic going to IGEL Community on Slack).

What are your thoughts on this?

**Comment by falkheiland** [comment]
This here is more of a forum style way - so you can find things too ;)
Also for creating / discussing features, issues etc this is a way more direct way - one i would prefer by a lot for those cases tbh.

I am sure there are / will be ways to integrate the discussion in slack via webhook though.

**Reply by falkheiland** [reply]
just activated the github integration of this repo to #github-planning in slack to check if discussions are in yet (i dont think so).

**Reply by falkheiland** [reply]
discussions not (yet) integrated in the slack app,
